### PR TITLE
should not proxy /keys commands to WDA if current context is web view

### PR DESCRIPTION
**Pull request by yaphatak**

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -93,5 +93,16 @@ commands.launchApp = iosCommands.general.launchApp;
 
 commands.closeApp = iosCommands.general.closeApp;
 
+commands.keys = async function (keys) {
+  if (!this.isWebContext()) {
+    throw new errors.UnknownError('Command should be proxied to WDA');
+  }
+  let el = await this.active();
```
Comment (jlipps) [inline lib/commands/general.js]: interesting, so there's no "send keys to active element" atom we can use directly? I find it odd we have to get the active element ourselves.

Comment (yaphatak) [inline lib/commands/general.js]: I got it from here https://github.com/appium/appium-ios-driver/blob/master/lib/commands/general.js#L113-L131
Thats how we have been doing it :)
I guess there is no keyboard sendkeys in `webview` world. Just guess!

Comment (yaphatak) [inline lib/commands/general.js]: I dont know of any active issue with this. This is something that I came across with our team.

**Comment by jlipps** [comment]
sounds good

**Pull request merged**